taskcluster/ci/signing/kind.yml

```diff
@@ -49,16 +42,41 @@ jobs:
     production:
         attributes:
             build-type: production
+        index:
+            type: signing
+        worker:
+            signing-type:
+                by-level:
+                    '3': production-signing
+                    default: dep-signing
         dependencies:
             build: 'Build FenixProduction task'  # comes from the old-decision task
         run-on-tasks-for: [github-release]
         treeherder:
             symbol: production(s)
 
+    fennec-production:
+        attributes:
+            build-type: fennec-production
+            nightly: true
+        worker:
+            signing-type: dep-signing   # TODO Use real fennec signature once mobile signing workers are ready
```

Comment:
Is this still a TODO? We already have builds with the right signatures right now? Or what does this mean? 🤔 

https://tools.taskcluster.net/index/project.mobile.fenix.v2.fennec-production/latest

Reply:
Great catch! I based my testing on tasks which were younger than the one you pointed out. There's been a regression since then! Fixed.